gk/main.c

```diff
@@ -31,6 +33,8 @@
 
 #define GATEKEEPER_MAX_PKT_BURST (32)
 
+static struct gk_config *gk_conf;
```

Comment:
The reason is that, the functions in GK block will all need the variable, so I tried to make it global within the block. So, the parameters of these functions seem simpler. Anyway, I will keep in mind that using global variables as less as possible.